apps/prairielearn/src/pages/instructorAssessmentStatistics/instructorAssessmentStatistics.html.ts

```diff
@@ -0,0 +1,333 @@
+                              return user.duration_secs;
+                            })}`,
+                          )},
+                        ];
+                        const ydata = [
+                          ${unsafeHtml(
+                            `${userScores.map((user) => {
+                              return user.score_perc;
+                            })}`,
+                          )},
+                        ];
+                        const options = {
+                          xgrid: [${unsafeHtml(`${durationStat.threshold_seconds}`)}],
+                          ygrid: _.range(0, 110, 10),
+                          xlabel: 'duration',
+                          ylabel: 'score / %',
+                          xTickLabels: [
+                            ${unsafeHtml(
+                              `${durationStat.threshold_labels.map(function (label) {
+                                return JSON.stringify(label);
+                              })}`,
+                            )},
+                          ],
+                        };
+                        scatter('#durationScoreScatter', xdata, ydata, options);
+                      });
+                    </script>
+                  </div>
+
+                  <div class="card-footer">
+                    <small>
+                      Each point is one student assessment. Points beyond the plot range are plotted
+                      on the edge.
+                    </small>
+                  </div>
+                `
+              : html`<div class="card-body">No student data.</div>`}
+          </div>
+
+          <div class="card mb-4">
+            <div class="card-header bg-primary text-white">
+              ${resLocals.assessment_set.name} ${resLocals.assessment.number}: Score statistics by
+              date
+            </div>
+
+            ${resLocals.assessment.score_stat_number > 0
+              ? html`
+                  <div class="card-body">
+                    <div
+                      id="scoreHistsByDateDiv"
+                      style="overflow-x: scroll; overflow-y: hidden;"
+                    ></div>
+                    <script>
+                      $(function () {
+                        var data = [
+                          ${unsafeHtml(
+                            `${assessmentScoreHistogramByDate.map(function (day) {
+                              return JSON.stringify({
+                                label: day.date_formatted,
+                                mean: day.mean_score_perc,
+                                histogram: day.histogram,
+                              });
+                            })}`,
+                          )},
+                        ];
+
+                        var options = {
+                          ygrid: _.range(0, 110, 10),
+                          xgrid: [${escapeHtml(html`${assessmentScoreHistogramByDate.length}`)}],
```

Comment:
Same comment as above.

Reply:
This has been moved to a data attribute in [fe614f8](https://github.com/PrairieLearn/PrairieLearn/pull/9809/commits/fe614f896046789604dac2796569dc2b7421cebc).